### PR TITLE
Add `xion-testnet-1` network info

### PR DESCRIPTION
**Pull request by froch**
- the Xion testnet is the first testnet from Burnt Labs
- https://xion.burnt.com

### CODE REVIEW

File: testnets/xiontestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "xion-testnet-1",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Xion Testnet",
+  "chain_id": "xion-testnet-1",
+  "bech32_prefix": "xion",
```
Comment (JeremyParish69) [inline testnets/xiontestnet/chain.json:8]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (froch) [reply testnets/xiontestnet/chain.json:8]: Thank you for pointing that out, we've opened a PR there.

https://github.com/satoshilabs/slips/pull/1531

Comment (froch) [reply testnets/xiontestnet/chain.json:8]: will set this PR as ready for review when the PR to satoshilabs goes through

**Comment by JeremyParish69** [comment]
Haven't seen any correction. Should we close this for now?

**Pull request closed without merging**